docs/source/opens-upgrade.rst

```diff
@@ -87,27 +87,16 @@ instead.)
 Ubuntu 14.04
 ^^^^^^^^^^^^
 
-First, upgrade packaged components::
+First, use ``apt-get`` to install the updated packages.  On each compute host
+you can upgrde only the Calico packages, as follows::
 
     apt-get update
-    apt-get dist-upgrade
+    apt-get install calico-compute calico-felix calico-common python-etcd
```

Comment:
Surely this is insufficient, as it misses lots of packages from the list below (being deleted).